src/test/java/com/serotonin/bacnet4j/obj/AccumulatorObjectTest.java

```diff
@@ -147,8 +143,8 @@ public void intrinsicReporting() throws Exception {
         a.writePropertyInternal(PropertyIdentifier.limitEnable, new LimitEnable(false, true));
         assertEquals(EventState.normal, a.readProperty(PropertyIdentifier.eventState));
         Thread.sleep(40);

```

Comment:
Review usage of thread.sleep

Reply:
Seems like we are awaiting a condition here no?

Reply:
Let's address this in a separate PR.